pkg/postgres/pg.go

```diff
@@ -8,20 +8,25 @@ import (
 	"time"
 )
 
+//go:generate go-bindata -prefix ../.. -pkg migrations -o ../migrations/migrations.go ../../migrations
+
 type PostgresStorage struct {
 	DB *sql.DB
 }
 
 func NewPostgresStorage(connect string) (*PostgresStorage, error) {
```

Comment:
You should pass db connect from outside. Don't open connect in Data Abstract Layer constructor.